drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/java/org/drools/workbench/screens/guided/dtable/client/widget/analysis/DecisionTableAnalyzer.java

```diff
@@ -146,19 +141,28 @@ protected AnalysisReport makeAnalysisReport() {
         return report;
     }
 
-    protected void sendReport( final AnalysisReport report ) {
-        IOC.getBeanManager().lookupBean( AnalysisReportScreen.class ).getInstance().showReport( report );
+    private void sendReport( final AnalysisReport report ) {
+        getAnalysisReportScreen().showReport( report );
+    }
+
+    private void sendStatus( final Status status ) {
+        getAnalysisReportScreen().showStatus( status );
+    }
+
+    protected AnalysisReportScreen getAnalysisReportScreen() {
+        return IOC.getBeanManager().lookupBean( AnalysisReportScreen.class ).getInstance();
```

Comment:
Ok. Made some changes to this area. There should be only a single instance of AnalysisReportScreen and the code should reuse it. I'll make sure there is one.